src/com/paynemiller/Runner.java

```diff
@@ -72,7 +76,7 @@ private static void startCharacterCounter() {
         try {
             counter.countCharacters(word);
         } catch (EmptyWordException e) {
-            e.printStackTrace(); // TODO: Gracefully handle exception.
```

Comment:
Are we still gonna handle the exception gracefully? If not, let's remote the comment.

Reply:
The comment has been removed.